packages/infoportal-client/src/shared/Datatable/util/generateEmptyXlsFile.ts

```diff
@@ -0,0 +1,69 @@
+import * as ExcelJS from 'exceljs'
+import {Utils} from '@/utils/utils'
+import {Kobo} from 'kobo-sdk'
+import Question = Kobo.Form.Question
+import Choice = Kobo.Form.Choice
+import {KoboSchemaHelper} from 'infoportal-common'
+
+const writeDropdownOptions = (
+  optionsSheet: ExcelJS.Worksheet,
+  header: string,
+  values: string[],
+  columnIndex: number
+): string => {
+  optionsSheet.getCell(1, columnIndex).value = header
+
+  values.forEach((value, rowIndex) => {
+    optionsSheet.getCell(rowIndex + 2, columnIndex).value = value
+  })
+
+  const startRow = 2
+  const endRow = values.length + 1
+  return `Options!$${String.fromCharCode(64 + columnIndex)}$${startRow}:$${String.fromCharCode(64 + columnIndex)}$${endRow}`
+}
+
+export const generateEmptyXlsTemplate = async (
+  schemaBundle: KoboSchemaHelper.Bundle,
+  fileName: string
+): Promise<void> => {
+  const workbook = new ExcelJS.Workbook()
+  const optionsSheet = workbook.addWorksheet('Options')
+  const templateSheet = workbook.addWorksheet('Template')
+
+  const questions = schemaBundle.schema.content.survey.filter(
+    (question: Question) =>
+      !['begin_group', 'end_group', 'begin_repeat', 'end_repeat'].includes(question.type)
```

Comment:
Not really meaningful since there are usually only hundreds of questions in a form, but just so you know. In this case, creating a `Set` head is more efficient
```
const exludedQuestions = new Set(['begin_group', 'end_group', 'begin_repeat', 'end_repeat'])
const questions = schemaBundle.schema.content.survey.filter(q => !exludedQuestions.has(q.type))
```